controllers/topics/index.js

```diff
@@ -180,7 +180,7 @@ const getTopics = async (req, res) => {
 const createTopic = async (req, res) => {
   try {
     let {name} = req.body;
-    name = name.toLowerCase().trim();
+    name = name.replace(/#/g, '').toLowerCase().trim();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change made to remove hashtags from the topic name is a good start, but it might be better to handle other special characters as well. This will ensure that the topic name is clean and doesn't contain any unexpected characters which could potentially cause issues later on.
```diff
-     name = name.replace(/#/g, '').toLowerCase().trim();
+     name = name.replace(/[^\w\s]/gi, '').toLowerCase().trim();
```
This new regex will replace all non-word characters (not a letter, number, or underscore) and spaces globally in the string.


<!-- This is an auto-generated comment by OSS CodeRabbit -->